behave/contrib/csv_table_from_file.py

```diff
@@ -1,43 +1,83 @@
-import logging
+# -*- coding: UTF -*-
```

Comment:
Mark this file as python3-only (if you want to use type annotations).
HINT: 
* `behave` supports currently still Python 2.7.
* Use type annotations causes annoying problems if Python 2.7 is still used.

Reply:
Resolved